Cargo.toml

```diff
@@ -14,6 +14,13 @@ homepage = "http://www.maidsafe.net"
 rand = "*"
 rust-crypto ="*"
 tempdir = "*"
-docopt_macros = "*"
-rustc-serialize = "*"
 
+[lib]
+name = "self_encryption"
+bench = false
+
+[[bin]]
```

Comment:
This is unnecessary, just put se.rs in /examples and then you can use `cargo run --example se -- args` (might be a little off since it's from memory but the gist of it should be correct)